packages/components/pie-toast/test/visual/pie-toast.spec.ts

```diff
@@ -1,118 +1,56 @@
-            isDismissible: ${testComponent.propValues.isDismissible},
-            isMultiline: ${testComponent.propValues.isMultiline},
-        `;
-
-        const darkMode = ['neutral-alternative'].includes(variant);
-
-        await mount(
-            WebComponentTestWrapper,
-            {
-                props: { propKeyValues, darkMode },
-                slots: {
-                    component: testComponent.renderedString.trim(),
-                },
-            },
-        );
-    }
-
-    await percySnapshot(page, `PIE Toast - Variant: ${variant}`, screenWidths);
-}));
-
 test.describe('Props', () => {
     test.describe('leadingAction', () => {
-        const props = {
-            message: 'Item was added',
-            leadingAction: {
-                text: 'Confirm',
-                ariaLabel: 'Button to confirm the action',
-            },
-        };
-        test('Displays leadingAction in footer if isMultiline is true', async ({ page, mount }) => {
-            await mount(PieToast, {
-                props: {
-                    ...props,
-                    isMultiline: true,
-                } as ToastProps,
-            });
+        test('should display leadingAction in footer if isMultiline is true', async ({ page }) => {
+            const toastPage = new BasePage(page, 'toast');
+            const props = {
+                message: 'Item was added',
+                isMultiline: true,
+                leadingAction: {
+                    text: 'Confirm',
+                    ariaLabel: 'Button to confirm the action',
+                },
+            };
+
+            await toastPage.load({ ...props });
 
-            await percySnapshot(page, 'PieToast - isMultiline - Displays leadingAction in footer if isMultiline is true');
+            await percySnapshot(page, 'PIE Toast - isMultiline - Displays leadingAction in footer if isMultiline is true', percyWidths);
         });
 
-        test('Displays leadingAction inline if isMultiline is false', async ({ page, mount }) => {
-            await mount(PieToast, {
-                props: {
-                    ...props,
-                    isMultiline: false,
-                } as ToastProps,
-            });
+        test('should display leadingAction inline if isMultiline is false', async ({ page }) => {
+            const toastPage = new BasePage(page, 'toast');
+            const props = {
+                message: 'Item was added',
+                isMultiline: false,
+                leadingAction: {
+                    text: 'Confirm',
+                    ariaLabel: 'Button to confirm the action',
+                },
+            };
+
+            await toastPage.load({ ...props });
 
-            await percySnapshot(page, 'PieToast - isMultiline - Displays leadingAction inline if isMultiline is false');
+            await percySnapshot(page, 'PIE Toast - isMultiline - Displays leadingAction inline if isMultiline is false', percyWidths);
         });
```

Comment:
I don't think we need these tests as they'll be covered by the first test when multiple variations are rendered. Please confirm and double check though!

Reply:
Yeah we still need these as the isMultiline with the leadingAction props are tested separately to the variations test. So either we add the leadingActions props into the variations tests (which will drastically increase the size of the screenshot) or we keep it as it is.

Reply:
I'd personally opt to add this to the visual tests then utilise the `multiColumn` functionality I added in button 👍🏼

Reply:
Running in the tests in 480px wide though so I don't think we can run the multiColumn.

Reply:
Update: Running our visual tests for toast at a wider viewport to make use of `multiColumn`.